src/context_free.ml

```diff
@@ -653,17 +653,17 @@ class map_top_down ?(expect_mismatch_handler = Expect_mismatch_handler.nop)
         x
 
     method! module_expr base_ctxt x =
-      ((* Make sure code-path attribute is applied before expanding. *)
-       Attribute.get_res Ast_traverse.enter_module x |> of_result ~default:None
-       >>= function
-       | None -> return (base_ctxt, x)
-       | Some { loc; txt } ->
-           Attribute.remove_seen_res Module_expr
-             [ T Ast_traverse.enter_module ]
-             x
-           |> of_result ~default:x
-           >>| fun x ->
-           (Expansion_context.Base.enter_module ~loc txt base_ctxt, x))
+      ( (* Make sure code-path attribute is applied before expanding. *)
+        Attribute.get_res Ast_traverse.enter_module x |> of_result ~default:None
+      >>= function
+        | None -> return (base_ctxt, x)
```

Comment:
Isn't the alignment of `>>= function` a little odd here?

Reply:
Anyway, nothing critical and I'm happy to merge those changes once 0.27.0 releases!